dap-etl/extraction/src/test/scala/org/broadinstitute/monster/dap/afus/AfusExtractionPipelineSpec.scala

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.monster.dap.afus
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import java.time.{OffsetDateTime, ZoneOffset}
+
+class AfusExtractionPipelineSpec extends AnyFlatSpec with Matchers {
+  it should "generate the correct arms when provided startTime and endTime arguments" in {
+    val startTime =
+      OffsetDateTime.of(2021, 10, 1, 0, 0, 0, 0, ZoneOffset.ofHours(-5))
+    val endTime =
+      OffsetDateTime.of(2023, 10, 1, 0, 0, 0, 0, ZoneOffset.ofHours(-5))
+
+    val afusArms = AfusExtractionPipeline.extractionArmsGenerator(Some(startTime), Some(endTime))
+
+    afusArms shouldBe List("fup_1_arm_1", "fup_2_arm_1", "fup_3_arm_1", "baseline_arm_1")
+  }
+
+  it should "use AFUSEpoch as startTime when none is provided" in {
+    val endTime =
+      OffsetDateTime.of(2022, 12, 1, 0, 0, 0, 0, ZoneOffset.ofHours(-5))
+    val afusArms = AfusExtractionPipeline.extractionArmsGenerator(None, Some(endTime))
+
+    afusArms shouldBe List("fup_1_arm_1", "fup_2_arm_1", "baseline_arm_1")
+  }
+
+  it should "create a list of arms inclusive of the last date in the range (endTime)" in {
```

Comment:
❓ Is this test the same as the one on L9?

Reply:
yeah it is basically testing the same thing as the first test in this file. removing this test.